openadapt/strategies/visual.py

```diff
@@ -382,6 +412,26 @@ def get_window_segmentation(
         # TODO XXX: create copy of similar_segmentation, but overwrite with segments of
         # regions of new image where segments of similar_segmentation overlap non-zero
         # regions of similar_segmentation_diff
+        new_image = vision.extract_difference_image(
```

Comment:
What do you think about naming this `difference_image` rather than `new_image`?

Reply:
makes sense, I'll change this